packages/expo-cli/jest/setup.ts

```diff
@@ -0,0 +1,2 @@
+// always mock:
+jest.mock('ora');
```

Comment:
Why do we even need to mock `ora`? Does it somehow adversely affect the test output, or something similar?

Reply:
Maybe you can eliminate the need for `setup.ts` by using https://jestjs.io/docs/en/configuration#modulenamemapper-objectstring-string--arraystring?

Reply:
Yeah, it clutters the test output. What's even worse, without mocking it, I'm almost sure we would see multiple lines printed on CircleCI.

Reply:
Regarding moduleNameMapper - I'll give it a try.

Reply:
Hmm, I wonder if that becomes a problem for our users too, in case they are running Expo CLI on CircleCI? Should we have a global option to disable it?

Reply:
That's a great question. Before doing anything, I'd check if the problem really exists. I think I saw it somewhere but maybe I'm mixing something.
At the same time - the real (/direct) reason why I added the mock for `ora` is that I came across it in the test suites added by Quin.

Reply:
Hmmm, I think I don't need `setup.ts` at all. It seems that if the __mocks__ directory is in the directory adjacent to node_modules then it's not necessary to call `jest.mock`.

Reply:
https://jestjs.io/docs/en/manual-mocks#mocking-node-modules says:
> If the module you are mocking is a Node module (e.g.: lodash), the mock should be placed in the __mocks__ directory adjacent to node_modules (unless you configured roots to point to a folder other than the project root) and will be automatically mocked. There's no need to explicitly call jest.mock('module_name').